### PR TITLE
increase threshold for heartbeat warnings

### PR DESCRIPTION
**Pull request by shkoder**
ref #25253

### CODE REVIEW

**Comment by johanandren** [comment]
OK TO TEST

File: akka-remote/src/main/scala/akka/remote/PhiAccrualFailureDetector.scala [hunk 1]
```diff
@@ -134,7 +134,7 @@ class PhiAccrualFailureDetector(
         val interval = timestamp - latestTimestamp
         // don't use the first heartbeat after failure for the history, since a long pause will skew the stats
         if (isAvailable(timestamp)) {
-          if (interval >= (acceptableHeartbeatPauseMillis / 2) && eventStream.isDefined)
+          if (interval >= (acceptableHeartbeatPauseMillis / 3 * 2) && eventStream.isDefined)
```
Comment (ktoso) [inline akka-remote/src/main/scala/akka/remote/PhiAccrualFailureDetector.scala]: Pretty arbitrary number – why?

Comment (ktoso) [reply akka-remote/src/main/scala/akka/remote/PhiAccrualFailureDetector.scala]: Ok, found the reason https://github.com/akka/akka/pull/25237#discussion_r196421342

**Review by ktoso — APPROVED** [review]

**Review by johanandren — APPROVED** [review]
LGTM

**Pull request merged**